Resources/Prototypes/Guidebook/shiftandcrew.yml

```
@@ -5,7 +5,11 @@
   children:
   - Engineering
   - Epistemics
+  - Chemistry #TODO replace with med
+<<<<<<< HEAD
+=======
 
+>>>>>>> ac73b1ab677079ccda328ec66ab6cc65f171ff5f
```

```suggestion
  - Chemistry #TODO replace with med
```
Same thing as before